### DISCUSSION TITLE
GitHub page not working

### DISCUSSION THREAD

**Discussion by CharlesLee01** [community/community#73112] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys,

Here's the link for my github pages code: https://github.com/CharlesLee01/react-page

The page was working fine a few days ago. However, now it just displays a blank screen and no new modifications have been made.

Please help me out ;-; Thanks a lot!

**Comment by ktauchathuranga** [comment]
console says
`Failed to load resource: the server responded with a status of 404 ()`
file `https://charleslee01.github.io/src/main.jsx` is missing
this might help you troubleshoot 🖤

**Reply by ktauchathuranga** [reply]
in your `index.html` file, in this line,
```html
<script type="module" src="/src/main.jsx"></script>
```
try changing that to,
```html
<script type="module" src="/react-page/src/main.jsx"></script>
```
to see if it works

**Reply by ivan-developer-01** [reply]
I'm sorry to say that won't work because JSX needs to be compiled...

**Reply by ktauchathuranga** [reply]
awwww....um sorry... and thanks for the update 🖤

**Comment by ivan-developer-01** [comment]
I think you should not connect jsx scripts directly. I think, you better run `npm run build` and upload the `build` folder to repository. Then, you will see it in the browser by visiting "https://charleslee01.github.io/react-page/dist/".


**Conclusion:** React pages cannot be run directly without specific server that supports React (GitHub Pages does not).

**Reply by CharlesLee01** [reply]
![image](https://github.com/community/community/assets/77286511/c31e2fe1-a8ec-4125-9231-49fc76436272)

Hmm... that's strange. I ran ```npm run deploy``` only because I have ```npm run build``` in package.json.

This problem appears on my react page with images only which were working fine a few days ago..

**Reply by ivan-developer-01** [reply]
If that helps, you can mark my comment as answer.

**Reply by CharlesLee01** [reply]
No.. its not working. I tried recreating the repo by removing the image and it works. Here's the working page https://charleslee01.github.io/react-page/

Once I add the image in, by uncommenting the image line in components/Navbar.jsx, the same problem occurs stating the main.jsx file cant be found. 

I didn't touch main.jsx at all though... ):

Can you fork it and try uncommenting it for me please. Tell me if it works <3

**Reply by ivan-developer-01** [reply]
Sorry it was my mistake. Can you try again `npm run build`, commit it, push it, and go to the URL `https://charleslee01.github.io/react-page/dist/`? Notice the `dist/` part.
